helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/ClusterAccessor.java

```diff
@@ -431,9 +426,104 @@ public Response getClusterStateModelDefinition(@PathParam("clusterId") String cl
     StateModelDefinition stateModelDef =
         dataAccessor.getProperty(dataAccessor.keyBuilder().stateModelDef(statemodel));
 
+    if (stateModelDef == null) {
+      return  notFound();
```

Comment:
Can we remove the extra space? It would be great to keep the code quality for open source :-)

Reply:
Used coding style format to correct it.